### ISSUE TITLE
Replace "open externally" button with "open internally" if browser is set to external

### ISSUE THREAD

**Issue by PopeRigby** [ccrama/Slide#3005]
There's currently no way to open the internal browser if your browser is set to external. This would allow you to do that.

**Comment by Tensounder54** [comment]
Opening in the internal browser is a pro feature so the it doesn't make a whole lot of sense. In the pro version, you can set it to open in reader view, the internal browser or the external browser. The only case where this might be relevant is with reader mode, auto-forwarding to the default browser if the reader mode rendering fails.

**Comment by lumbo7332** [comment]
Internal browser isn't just a pro feature

**Comment by Tensounder54** [comment]
Yes it is. The pro version is released on F-Droid for free as it's the only way that we can release on that platform.
